handbook/engineering/deployments/debugging/index.md

```diff
@@ -0,0 +1,15 @@
+# Debugging
+
+In order to debug the services we deploy you should understand the environment they are deployed on.
+
+This guide will allow you to debug one of our running services in the **dogfood** deployment which mimics our customer environment. All of our services here run on a Kubernetes cluster.
```

Comment:
Link to the k8s entry in the instances doc maybe?

Reply:
Good idea